webapp/src/main/java/com/box/l10n/mojito/quartz/QuartzConfig.java

```diff
@@ -32,15 +33,21 @@ public class QuartzConfig {
   @Autowired(required = false)
   List<JobDetail> jobDetails = new ArrayList<>();
 
+  @Autowired QuartzPropertiesConfig quartzPropertiesConfig;
+
   /**
    * Starts the scheduler after having removed outdated trigger/jobs
    *
    * @throws SchedulerException
    */
   @PostConstruct
   void startScheduler() throws SchedulerException {
+    Properties quartzProps = quartzPropertiesConfig.getQuartzProperties();
```

Comment:
For now, this will do but I find it weird to have to inject/read the properties here again vs. being able to read information on the scheduler instance itself. With the multi scheduler implementation we may have a new Scheduler class that could include a field like "shouldStart". And this QuartzConfig may be totally different anyway as the job detail and outdated jobs will probably be per scheduler instances.